diff -u -r nethogs-0.8.6/src/cui.cpp nethogs-0.8.6/src/cui.cpp
--- nethogs-0.8.6/src/cui.cpp	2020-02-19 00:35:13.000000000 +0800
+++ nethogs-0.8.6/src/cui.cpp	2023-04-28 20:38:03.000000000 +0800
@@ -374,7 +374,7 @@
   int totalrow = std::min(rows - 1, 3 + 1 + i);
   mvprintw(totalrow, 0, "  TOTAL        %-*.*s %-*.*s    %11.3f %11.3f ",
            proglen, proglen, "", devlen, devlen, "", sent_global, recv_global);
-  mvprintw(3 + 1 + i, cols - COLUMN_WIDTH_UNIT, desc_view_mode[viewMode]);
+	mvprintw(3 + 1 + i, cols - COLUMN_WIDTH_UNIT, "%s", desc_view_mode[viewMode]);
   attroff(A_REVERSE);
   mvprintw(totalrow + 1, 0, "");
   refresh();
